pkg/pmem-device-manager/pmd-lvm.go

```diff
@@ -287,3 +295,86 @@ func getVolumeGroups(groups []string) ([]vgInfo, error) {
 
 	return vgs, nil
 }
+
+func createNS(r *ndctl.Region, uselimit uint) error {
+	const align uint64 = 1024 * 1024 * 1024
+	realalign := align * r.InterleaveWays()
+	// uselimit is the percentage we can use
+	canUse := uint64(uselimit) * r.Size() / 100
+	klog.V(3).Infof("Create fsdax-namespaces in %v, allowed %d %%, real align %d:\ntotal       : %16d\navail       : %16d\ncan use     : %16d",
+		r.DeviceName(), uselimit, realalign, r.Size(), r.AvailableSize(), canUse)
+	// Subtract sizes of existing active namespaces with currently handled mode and owned by pmem-csi
+	for _, ns := range r.ActiveNamespaces() {
+		klog.V(5).Infof("createNS: Exists: Size %16d Mode:%v Device:%v Name:%v", ns.Size(), ns.Mode(), ns.DeviceName(), ns.Name())
+		if ns.Name() == "pmem-csi" {
+			canUse -= ns.Size()
```

Comment:
Are you absolutely sure that the uint64 cannot underflow?

At some point in the past, the percentage may have been higher and a volume might have been created which is higher than the currently allowed maximum.

Reply:
Fixed this case.